4-Express/client/scripts/communication_manager.js

```diff
@@ -0,0 +1,38 @@
+const axios = require("axios").default;
+const SERVER_URL = "http://localhost:3000";
+
+function addOrUpdateItem(id, item) {
+  const urlWithId = `${SERVER_URL}/${id}`;
+
+  axios.post(urlWithId, item).then(
+    (response) => {
+    },
+    (error) => {
+      console.log(error);
+    }
+  );
+}
+
+function removeItem(id) {
+  const urlWithId = `${SERVER_URL}/${id}`;
+
+  axios.delete(urlWithId, { params: { id: id } }).then(
+    (response) => {
+    },
+    (error) => {
+      console.log(error);
+    }
+  );
+}
+
+async function getItems() {
+  try {
+    const response = await axios.get(SERVER_URL);
+    return response.data;
```

Comment:
remember that `await` is blocking the execution so `line 31` will be delayed until the http request is resolved. 
consider returning the promise and let the consumer decide what to do with it